lib/Parser/OglParser.php

```diff
@@ -23,32 +23,34 @@
 class OglParser
 {
     private $hints;
+    private $matrix;
 
-    public function __construct($hints) {
+    public function __construct($hints, $matrix) {
         $this->hints = $hints;
+        $this->matrix = $matrix;
     }
 
-    public function parse($filename) {
+    public function parse($filename, $commit = null) {
         $handle = fopen($filename, "r");
```

Comment:
@MightyCreak Hehe, if this gets any bigger *then* we can worry about unit testing and the like.